clusterloader/framework/util.go

```diff
@@ -80,3 +113,86 @@ func CreatePods(f *framework.Framework, appName string, ns string, labels map[st
 		}
 	}
 }
+
+func CreateRC(f *framework.Framework, namespace, name, imageName string, label map[string]string, replicas int) error {
+	nodeSelector := fields.OneTermEqualSelector("metadata.name", name)
+	list, _ := f.ClientSet.Core().ReplicationControllers(namespace).List(metav1.ListOptions{FieldSelector: nodeSelector.String()})
+	switch len(list.Items) {
+	case 0:
+		newRC := newRC(name, int32(replicas), label, name, imageName)
+		rc, err := f.ClientSet.Core().ReplicationControllers(namespace).Create(newRC)
+		if err != nil {
+			return err
+		}
+		framework.Logf("NEW RC: %+v", rc)
+	case 1:
+		rc, err := framework.UpdateReplicationControllerWithRetries(f.ClientSet, namespace, name, func(update *v1.ReplicationController) {
+			x := int32(replicas)
+			update.Spec.Replicas = &x
+		})
+		if err != nil {
+			return err
+		}
+		framework.Logf("UPDATED RC: %+v", rc)
```

Comment:
k8s doesn't support having duplicate named objects so it would only return 0 or 1. Will change 1 to default.